### PR TITLE
Apps define as relative path

### PR DESCRIPTION
**Pull request by voatsap**
- Move application definition to relative path
- Enhance documentation about it:
https://github.com/shalb/cluster.dev/tree/apps-as-relative-path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,59 +2,138 @@
...
+Required Managed policies: _AmazonEC2FullAccess, AmazonS3FullAccess, AmazonRoute53FullAccess, AmazonDynamoDBFullAccess_, or you can add permissions using this [iam-json](https://gist.github.com/dgozalo/bc4b932d51f22ca5d8dad07d9a1fe0f2).
 
+Resulting access pair should look like:
 ```yaml
 aws_access_key_id = ATIAAJSXDBUVOQ4JR
 aws_secret_access_key = SuperAwsSecret
 ```
 
-3. Add credentials to you repo Secrets under GitHub's: "Settings->Secrets":
+3. Add credentials to you repo Secrets under GitHub's repo setting: _"Settings->Secrets"_:
```
Comment (MaxymVlasov) [inline README.md]:
```suggestion
3. Add credentials to you repo Secrets under GitHub's repo setting: `Settings->Secrets`:
```

File: .cluster.dev/minikube-one.yaml [hunk 2]
```diff
@@ -5,14 +5,15 @@ cluster:
...
+# 'domain' you need to create/have it in your AWS Route53 account
+    domain: cluster.dev
   provisioner:
     type: minikube
     instanceType: m5.large
   modules:
     nginx-ingress: true
     cert-manager: true
-# 'apps' are actually directories in that contains ArgoCD project and application declarations
+# 'apps' are list of directories in you repo that contains ArgoCD applications and project manifests
```
Comment (anastyakulyk) [inline .cluster.dev/minikube-one.yaml]:
```suggestion
# 'apps' are the list of directories in you repo that contains ArgoCD applications and project manifests
```

File: README.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 Cluster.dev is an open-source system delivered as GitHub Action or Docker Image for creating and managing Kubernetes clusters with simple manifests by GitOps approach.
 
-Designed for developers that are bored to configure Kubernetes stuff and just need: kubeconfig, dashboard, logging and monitoring out-of-the-box.
+Designed for developers that are bored to configure Kubernetes stuff and just need: kubeconfig, dashboard, logging and monitoring and deployment systems out-of-the-box.
 
-Based on DevOps and SRE best-practices. GitOps cluster management and application delivery. Simple CI/CD integration. Easily extendable by pre-configured applications and modules. Supports different Cloud Providers and Kubernetes versions.
+GitOps infrastructure management with Terraform and continuous deployment with ArgoCD. Easily extendable by pre-configured applications and modules. Quick integration with Jenkins, GitLab or  other CI/CD systems. Supports multiple Cloud Providers and Kubernetes versions.
```
Comment (anastyakulyk) [inline README.md]:
```suggestion
Uses GitOps infrastructure management with Terraform and continuous deployment with ArgoCD. Easily extendable by pre-configured applications and modules. Allows for quick integration with Jenkins, GitLab or other CI/CD systems. Supports multiple Cloud Providers and Kubernetes versions.
```

File: README.md [hunk 4]
```diff
@@ -25,22 +25,27 @@ Based on DevOps and SRE best-practices. GitOps cluster management and applicatio
...
-This repo will host code for your clusters, deployments and other resources managed by GitOps.  
-Next steps should be done in that repo.
+1. Dedicate a separate repository for the infrastructure code that will be managed by `cluster.dev`.  
+This repo will host code for your clusters, deployments, applications and other resources.  
+So, next steps should be done in that repo.
 
-2. Obtain access credentials for your non-root cloud account.  
-In AWS it is called "Programmatic Access user", and looks like:
+2. Create credentials for your non-root cloud account.  
+In AWS you need to use existing or create new ["Programmatic Access user"](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console).
```
Comment (anastyakulyk) [inline README.md]:
```suggestion
In AWS you need to use the existing or create the new ones ["Programmatic Access user"](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console).   
```

File: README.md [hunk 5]
```diff
@@ -25,22 +25,27 @@ Based on DevOps and SRE best-practices. GitOps cluster management and applicatio
...
 
-3. Add credentials to you repo Secrets under GitHub's: "Settings->Secrets":
+3. Add credentials to you repo Secrets under GitHub's: _"Settings->Secrets"_:
  ![GitHub Secrets](docs/images/gh-secrets.png)
 
-4. Create a Github Workflow file and cluster.dev manifest with your cluster definition.
+4. In your repo, create a Github Workflow file: [.github/workflows/main.yml](.github/workflows/main.yml) and  
+ cluster.dev example manifest: [.cluster.dev/minikube-one.yaml](.cluster.dev/minikube-one.yaml) with cluster definition.
+
+_Also example files could be pulled and placed to your repo with the next commands:_
```
Comment (anastyakulyk) [inline README.md]:
```suggestion
_Also example files could be pulled and placed to your repo with the following commands:_
```

File: README.md [hunk 6]
```diff
@@ -50,11 +55,20 @@ mkdir -p .github/workflows/ && wget -O .github/workflows/main.yml "https://raw.g
...
 
-5. In cluster definition you should set your own Route53 zone (create it manually, if needed). After implementation of [#2](https://github.com/shalb/cluster.dev/issues/2) this step will not be required.  
-And you can change all other parameters or leave default values.  
+5. In cluster definition yaml you should set your own Route53 DNS zone. If you don't have any hosted public zone you can create it manually with [instructions from AWS Website](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html).  
+
+6. You can change all other parameters or leave default values in cluster.yaml .  
 Leave github workflow file as is.
 
-6. Commit and Push files to your repo and follow the Github Action execution status. In GitHub action output you'll receive access instructions to your cluster and services.
+7. Copy sample ArgoCD applications from [/kubernetes/apps/samples](https://github.com/shalb/cluster.dev/tree/master/kubernetes/apps/samples) and Helm chart samples from [/kubernetes/charts/wordpress](https://github.com/shalb/cluster.dev/tree/master/kubernetes/charts/wordpress) to the same paths into your repo and define path Apps in cluster manifest:
```
Comment (anastyakulyk) [inline README.md]:
```suggestion
7. Copy a sample of ArgoCD applications from [/kubernetes/apps/samples](https://github.com/shalb/cluster.dev/tree/master/kubernetes/apps/samples) and Helm chart samples from [/kubernetes/charts/wordpress](https://github.com/shalb/cluster.dev/tree/master/kubernetes/charts/wordpress) to the same paths into your repo and define path Apps in cluster manifest:
```

File: README.md [hunk 7]
```diff
@@ -2,59 +2,138 @@
 
 Cluster.dev is an open-source system delivered as GitHub Action or Docker Image for creating and managing Kubernetes clusters with simple manifests by GitOps approach.
 
-Designed for developers that are bored to configure Kubernetes stuff and just need: kubeconfig, dashboard, logging and monitoring out-of-the-box.
+Designed for developers that are bored to configure Kubernetes stuff and just need: kubeconfig, dashboard, logging and monitoring and deployment systems out-of-the-box.
 
-Based on DevOps and SRE best-practices. GitOps cluster management and application delivery. Simple CI/CD integration. Easily extendable by pre-configured applications and modules. Supports different Cloud Providers and Kubernetes versions.
+GitOps infrastructure management with Terraform and continuous deployment with ArgoCD. Easily extendable by pre-configured applications and modules. Quick integration with Jenkins, GitLab or  other CI/CD systems. Supports multiple Cloud Providers and Kubernetes versions.
```
[the remark this answers is not in the document]
Comment (voatsap) [reply README.md]: Gitops Infrastructure management - not infrastructure

File: README.md (hunk 3 above)
Comment (voatsap) [reply README.md]: GitOps infrastructure management - means infrastructure management in gitops style

File: README.md [hunk 8]
```diff
@@ -2,59 +2,138 @@
...
 ----
 
 ## MENU
-
-* [Quick Start](#quick-start-)
-  * [Quick Start on AWS](#quick-start-on-aws-)
-  * [Cleanup](#cleanup-)
 * [Principle diagram](#principle-diagram-)
 * [How it works](#how-it-works-)
+* [Installation](#installation-)
```
[the remark this answers is not in the document]
Comment (voatsap) [reply README.md]: I'm using most common usage words. Lets keep it understendable not formal. 
Re-think this in next iteration

File: README.md [hunk 9]
```diff
@@ -2,59 +2,138 @@
...
 * [Roadmap](#roadmap-)
 * [Contributing](#contributing-)
 
 ----
 
-## Quick Start [`↑`](#menu)
+
+## Principle diagram [`↑`](#menu)
+
+Small and understandable yaml definition:
```
[the remark this answers is not in the document]
Comment (voatsap) [reply README.md]: Don't get this

File: README.md [hunk 10]
```diff
@@ -2,59 +2,138 @@
...
+Required Managed policies: _AmazonEC2FullAccess, AmazonS3FullAccess, AmazonRoute53FullAccess, AmazonDynamoDBFullAccess, IAMFullAccess, AmazonEC2ContainerRegistryReadOnly, ElasticLoadBalancingFullAccess_, or you can add permissions using this [iam-json](https://gist.github.com/dgozalo/bc4b932d51f22ca5d8dad07d9a1fe0f2).
 
+Resulting access pair should look like:
 ```yaml
 aws_access_key_id = ATIAAJSXDBUVOQ4JR
 aws_secret_access_key = SuperAwsSecret
 ```
 
-3. Add credentials to you repo Secrets under GitHub's: "Settings->Secrets":
+3. Add credentials to you repo Secrets under GitHub's repo setting: `Settings->Secrets`:
```
Comment (MaxymVlasov) [inline README.md]:
```suggestion
3. Add credentials to you repo Secrets under GitHub's repo setting: `Settings → Secrets`:
```

File: README.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (voatsap) [reply README.md]: Its like visual eye-catcher, on benefits of usage.

**Pull request merged**